app/templates/admin.hbs

```diff
@@ -0,0 +1,51 @@
+{{#if session.isAuthenticated}}
+
+  {{paper-input label='Search books' value=search onChange=(action (mut search))}}
+
+  {{#if (can "search books")}}
```

Comment:
If you have if after if you can usually combine them and use and helper for instance or convert it to computed property and give it a proper name and hide the logic behind the computed property.

Reply:
Both `if` statements have corresponding `else` block statements and they are displaying separate segments of UI so it will be hard to combine, and wrap them into helper or computed property.